docs/loading/Flink-connector-starrocks.md

```diff
@@ -1,173 +1,647 @@
+The Flink connector supports DataStream API, Table API & SQL, and Python API. The Flink connector also has a higher performance than [flink-connector-jdbc](https://nightlies.apache.org/flink/flink-docs-master/docs/connectors/table/jdbc/) provided by Apache Flink®.
 
-This topic describes how to load data from Apache Flink® to StarRocks.
+## Version requirements
 
-<InsertPrivNote />
+| Connector | Flink                    | StarRocks     | Java | Scala     |
+|-----------|--------------------------|---------------| ---- |-----------|
+| 1.2.7     | 1.11,1.12,1.13,1.14,1.15 | 2.1 and later | 8    | 2.11,2.12 |
 
-## Overview
+## Obtain Flink connector
 
-The flink-connector-jdbc tool provided by Apache Flink® may not meet your performance requirements in certain scenarios. Therefore we provide a new connector named flink-connector-starrocks, which can cache data and then load data at a time by using [Stream Load](./StreamLoad.md).
+You can obtain the Flink connector JAR file in the following ways:
 
-## Procedure
+- Directly download the compiled Flink Connector JAR file.
+- Add the Flink connector as a dependency in your Maven project and then download the JAR file.
+- Compile the source code of the Flink Connector into a JAR file by yourself.
 
-To load data from Apache Flink® into StarRocks by using flink-connector-starrocks, perform the following steps:
+The naming format of the Flink connector JAR file
 
-1. Download the [source code](https://github.com/StarRocks/flink-connector-starrocks) of flink-connector-starrocks.
-2. Find a file named **pom.xml**. Add the following code snippet to **pom.xml** and replace `x.x.x` in the code snippet with the latest version number of flink-connector-starrocks.
+- Since Flink 1.15, it's `flink-connector-starrocks-${connector_version}_flink-${flink_version}.jar`. For example, if you install Flink 1.15 and you want to use Flink connector 1.2.7, you can use `flink-connector-starrocks-1.2.7_flink-1.15.jar`
 
-    ```Plain%20Text
+- Prior to Flink 1.15, it's `flink-connector-starrocks-${connector_version}_flink-${flink_version}_${scala_version}.jar`. For example, if you install Flink 1.14 and Scala 2.12 in your environment, and you want to use Flink connector 1.2.7, you can use `flink-connector-starrocks-1.2.7_flink-1.14_2.12.jar`.
+
+> **NOTICE**
+>
+> In general, the latest version of the Flink connector only maintains compatibility with the three most recent versions of Flink.
+
+### Download the compiled Jar file
+
+Directly download the corresponding version of the Flink connector JAR from the [Maven Central Repository](https://repo1.maven.org/maven2/com/starrocks).
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/aecf8529-41b9-44f7-9002-e8e6b21ad5ab)
Add "file" following "JAR". It should be "JAR file".